share/applications/kakoune.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Version=
```

Comment:
The spec says:

> Version of the Desktop Entry Specification that the desktop entry conforms with. Entries that confirm with this version of the specification should use `1.0`. Note that the version field is not required to be present.

So this key should probably set to `1.0`, or removed entirely, rather than being left blank.